evalml/tests/automl_tests/test_iterative_algorithm.py

```diff
@@ -327,23 +433,34 @@ def test_iterative_algorithm_passes_n_jobs_catboost_xgboost(
 
 @pytest.mark.parametrize("ensembling_value", [True, False])
 def test_iterative_algorithm_one_allowed_pipeline(
-    ensembling_value, logistic_regression_binary_pipeline_class
+    X_y_binary, ensembling_value, dummy_binary_pipeline_classes
 ):
+    X, y = X_y_binary
+
+    (
+        dummy_binary_pipeline_classes,
+        allowed_component_graphs,
+    ) = dummy_binary_pipeline_classes()
+    dummy_binary_pipeline_classes = [dummy_binary_pipeline_classes[0]]
+    allowed_component_graphs = {"graph_1": allowed_component_graphs["graph_1"]}
```

Comment:
Nitpick: I saw that you added `logistic_regression_binary_component_graph`, could be used here instead w/ keeping the original `logistic_regression_binary_pipeline_class`? Just noticed since you're taking the fixture and only taking the first element anyways.